assets/app/scripts/directives/logViewer.js

```diff
@@ -275,6 +275,10 @@ angular.module('openshiftConsole')
                 $timeout(function() {
                   $scope.loading = false;
                 }, 100);
+
+                if(lastLineNumber === 0) {
+                    $scope.emptyState = true;
```

Comment:
Agree.  Working on that.  

It seems to work fine out side the `$evalAsync`, but since we have that block here anyway, I'll move this up inside it.